app/genesis.go

```diff
@@ -105,8 +105,17 @@ func NewDefaultGenesisState() GenesisState {
 			SendEnabled: true,
 		},
 		MintData: mint.GenesisState{
+			Minter: mint.Minter{
+				Inflation: sdk.NewDecWithPrec(13, 2),
+				AnnualProvisions: sdk.NewDec(0),
+			},
 			Params: mint.Params{
-				TokensPerBlock: 634195840,
+				MintDenom:           coin.CYB,
+				InflationRateChange: sdk.NewDecWithPrec(13, 2),
+				InflationMax:        sdk.NewDecWithPrec(20, 2),
+				InflationMin:        sdk.NewDecWithPrec(7, 2),
```

Comment:
Due to my proposal of higher bond target (90%) I think min inflation must be lower, especially because in the first 500 days of the network some supply will be locked in auction. Hence 5%

Reply:
Do I understand correctly that the minimum value is 5%, the maximum is 20%, the maximum change is 13%? Bond target - 90%?